components/__snapshots__/quick_input.test.jsx.snap

```diff
@@ -0,0 +1,199 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/QuickInput should render clear button with customized tooltip component 1`] = `
+<div
+  className="input-clear visible"
+  onClick={[Function]}
+>
+  <OverlayTrigger
+    defaultOverlayShown={false}
+    delayShow={400}
+    overlay={
+      <Tooltip
+        bsClass="tooltip"
+        id="InputClearTooltip"
+        placement="right"
+      >
+        <span>
+          Custom
+        </span>
+      </Tooltip>
+    }
+    placement="bottom"
+    trigger={
+      Array [
+        "hover",
+        "focus",
+      ]
+    }
+  >
+    <OverlayTrigger
```

Comment:
This double nesting surprised me, but it's not an artifact of my changes. It seems to just be how we propagate the props through to `BaseOverlayTrigger`.